servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -424,18 +423,16 @@ public void moveTempBlockTest() throws Exception {
     mBlockStore.moveBlock(SESSION_ID1, TEMP_BLOCK_ID, mTestDir2.toBlockStoreLocation());
   }
 
-  // TACHYON-825
-  @Ignore
   @Test
-  public void moveBlockToTheLocationWithExistingIdTest() throws Exception {
+  public void cacheSameBlockInDifferentTiersTest() throws Exception {
```

Comment:
Turns out this was the test case (both mTestDir1 and mTestDir2 are in the same tier). I added another test that uses mTestDir1 and mTestDir3 that are in different tiers.